src/rgw/rgw_tracer.h

```diff
@@ -0,0 +1,77 @@
+#ifndef TRACER_WRAPPER_H
+#define TRACER_WRAPPER_H
+
+#define SIGNED_RIGHT_SHIFT_IS 1
+#define ARITHMETIC_RIGHT_SHIFT 1
+#include <jaegertracing/Tracer.h>
+#include <yaml-cpp/yaml.h>
```

Comment:
this is probably only needed in the cc file

Reply:
I agree about the YAML include.
the rest of them are needed in the header file.